ansible/roles/debops.postldap/defaults/main.yml

```diff
@@ -19,6 +19,7 @@
 # List of additional APT packages needed for the LDAP support.
 postldap__postfix__dependent_packages: '{{ [ "postfix-ldap" ]
                                            if postldap__ldap_enabled else [] }}'
+
                                                                    # ]]]
                                                                    # ]]]
```

Comment:
The secdion endings are exception to the rule - since you already have two lines here, the spacing between variables is fine, and you shouldn't add additional blank space. Now it will look "off" in the other direction. Sorry for pedantry. :-)